test/expressions/generateExpressionTests.py

```diff
@@ -14,6 +14,10 @@
 exceptions_list_location = (
     "./test/expressions/stan_math_sigs_exceptions.expected"
```

Comment:
I think add these function exceptions to `./test/expressions/stan_math_sigs_exceptions.expected`

No need to make a separate list.

Reply:
I actually asked @t4c1 about this one and he suggested to go this route. Mainly for two reasons: 
- the exception lists carries the list of sigs that dont support expressions. lmultiply does support it, its just different in that is has no direct Math implementation.Analysis_and_optimization-
- the plan is to eventually get rid of the file by supporting expressions everywhere

Reply:
I could still be persuaded otherwise. Just think those are both valid points.

Reply:
I still think stick with the single file for exceptions.

I think there are already functions not in stan math that are in there: https://github.com/stan-dev/math/blob/develop/test/expressions/stan_math_sigs_exceptions.expected#L10

> the plan is to eventually get rid of the file by supporting expressions everywhere

I think we just change the meaning of this file to be all the signatures we can't test in Math because either they are compatible (and this meaning will go away) or they don't exist (cause they're Stan things)

Reply:
Ok, will add the sigs to the file and we can change the meaning once we clean out the non-compatible ones.

The sigs you linked are actually a bug and should not be listed: https://github.com/stan-dev/stanc3/issues/693